lib/drivers/abbott/abbottFreeStyleLite.js

```diff
@@ -357,30 +357,21 @@ module.exports = function (config) {
   };
 
   var setDateTime = function (serverTime, cb) {
-    var cmd = {
-      packet: struct.packString('$tim,' +
+    var cmd = struct.packString('$tim,' +
         sundial.formatInTimezone(serverTime, cfg.timezone, 'MM,DD,YY,HH,mm') +
-        '\r\n'),
-      parser: function (result) {
-        return result;
-      }
-    };
+        '\r\n');
 
-    freeStyleCommandResponse(cmd, function (err, result) {
+    serialDevice.writeSerial(cmd, function (err) {
       if (err) {
-        debug('Failure trying to talk to device.');
-        debug(err);
-        debug(result);
         cb(err, null);
-      } else {
-        debug('Device time set');
-        cb(null);
       }
+      console.log('Device time set');
```

Comment:
Should this be `debug`?

Reply:
Yes, you're right. I'm not going to make the change the right now, as that would mean another re-build and QA before we can push to prod, and this is not user-facing. But I will fix it in another branch to get merged into `develop`.